src/DataConvert.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace WPML\PB\Elementor;
+
+class DataConvert {
+
+	/**
+	 * @param array $data
+	 *
+	 * @return string
+	 */
+	public static function serialize( array $data ) {
+		return wp_slash( wp_json_encode( $data ) );
+	}
+
+	/**
+	 * @param array|string $data
+	 *
+	 * @return array
+	 */
+	public static function unserialize( $data ) {
+		$convertedData = $data;
+		if ( is_array( $data ) ) {
+			$convertedData = $data[0];
+		}
+
+		return json_decode( $convertedData, true );
```

Comment:
@strategio How about:
```php
return json_decode( is_array( $data ) ? $data[0] : $data, true );
```

Reply:
I just extracted existing code @brucepearson. But OK, let's make it simple.